packages/form-js-playground/karma.conf.js

```diff
@@ -48,7 +48,11 @@ module.exports = function(karma) {
         modules: [
           'node_modules',
           __dirname
-        ]
+        ],
+        alias: {
+          'react': 'preact/compat',
+          'react-dom': 'preact/compat'
+        }
```

Comment:
For what do we need this?

Reply:
We added SVGs to the form viewer, which also cascades into the forms playground. The SVG parser needs some kind of compatibility aliases to work, as it's been done in the form-editor already. Why this wasn't an issue already because of the form-editor, I'm not so sure, but I haven't been able to build without this.